expression/generator/compare_vec.go

```diff
@@ -251,6 +341,13 @@ func generateDotGo(fileName string, compares []CompareContext, types []TypeConte
 				if err != nil {
 					return err
 				}
+			} else if compareCtx.CompareName == "Coalesce" {
```

Comment:
For `Coalesce`, this is right.
But for `NullEQ` and `ET/GE...`, we want to generate all types except int type and write int type by hands, so here we need to continue if we meet int type.

Reply:
understood